### DISCUSSION TITLE
ublacklist-subscription-compilation/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#135] [Giscus]
# ublacklist-subscription-compilation/

这是一个uBlacklist插件的订阅地址合集，用于屏蔽中文SEO垃圾站，如机器翻译、AI生成的内容。通过Github Actions每周自动更新。插件可防止指定的网站出现在搜索引擎结果中，可以添加规则或点击工具栏图标屏蔽指定的网站。规则可使用匹配模式或正则表达式指定，并可通过云存储跨设备同步。支持的搜索引擎有Google、Bing、Brave、DuckDuckGo、Ecosia、Qwant、SearX、Startpage、Yahoo! JAPAN和Yandex。安装插件后，设置订阅黑名单列表并添加订阅地址。还可进行其他设置，如在谷歌搜索结果中屏蔽特定网站。

https://www.eallion.com/ublacklist-subscription-compilation/

**Comment by rebron1900** [comment]
好用。

**Reply by eallion** [reply]
还可以。搜索结果可以看了。

**Comment by flashlab** [comment]
很想屏蔽csdn，但有时候确实只有这里有有想要的🌝

**Reply by eallion** [reply]
确实有时候需要做取舍。

**Comment by ghost** [comment]
蜗牛大神，`jimowang.com` 的正则好像有点问题，应该是 `*://*.jimowang.com/*` 而不是 `*:/*.jimowang.com/*`，已经在 GitHub 上提 PR 了。

**Reply by ghost** [reply]
抱歉，我似乎有点理解错了。这个应该向规则的原始提供者提 PR。

**Reply by eallion** [reply]
https://github.com/eallion/uBlacklist-Subscription/commit/f96e8f7b0f74edc01a2161f56a2561d3549fa42b 处理了。
